app/src/Twig/AppExtension.php

```diff
@@ -9,17 +9,22 @@
 use App\Service\MessageService;
 use Doctrine\ORM\EntityManagerInterface;
 use Symfony\Bundle\SecurityBundle\Security;
+use Symfony\Component\Asset\Packages;
 use Symfony\Component\Security\Core\Authentication\Token\SwitchUserToken;
 use Twig\Extension\AbstractExtension;
 use Twig\TwigFilter;
 use Twig\TwigFunction;
+use App\Entity\Domain;
 
 class AppExtension extends AbstractExtension
 {
     public function __construct(
         private EntityManagerInterface $em,
         private MessageService $messageService,
         private Security $security,
+        private string $uploadDirectory,
+        private string $defaultLogoFilename,
+        private Packages $packages, // auto-wired apparently
```

Comment:
> // auto-wired apparently

Oui tout à fait, au même titre que les trois premiers arguments du constructeur d’ailleurs :). T'as de la doc sur comment ça fonctionne ici : https://symfony.com/doc/current/service_container.html

C’est une bonne question, ceci dit y a pas de raison d’y mettre un commentaire (sinon on finirait par en mettre littéralement partout en Symfony 🙃 )

```suggestion
        private Packages $packages,
```

Mais je m’en occupe en mergeant, t’embête pas pour ça !